_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -129,6 +138,16 @@ This `object` element which embeds an audio resource has an empty [accessible na
 <object aria-labelledby="download" data="/test-assets/moon-audio/moon-speech.mp3"></object>
 ```
 
+#### Failed Example 5
+
+This `object` element has an empty [accessible name][]. The `img` element inside the `object` is not used in computing the `object`'s accessible name.
+
+```html
+<object data="/test-assets/shared/w3c-logo.png">
+	<img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
+</object>
+```
+
```

Comment:
````suggestion

#### Failed Example 6

This `object` element has `alt` attribute, however this will not create an  [accessible name][]. Because of this the accessible name is empty.

```html
<object data="/test-assets/moon-audio/moon-speech.mp3" alt="JFK's Going to the moon speech"></object>
```

````